datastore_mongo.go

```diff
@@ -79,6 +79,21 @@ func NewDataStoreMongo(host string) (*DataStoreMongo, error) {
 	return db, nil
 }
 
+func (db *DataStoreMongo) GetDevices(skip, limit uint) ([]Device, error) {
+	s := db.session.Copy()
+	defer s.Close()
+
+	c := s.DB(DbName).C(DbDevicesColl)
+
+	res := []Device{}
+
+	err := c.Find(bson.M{}).Skip(int(skip)).Limit(int(limit)).All(&res)
```

Comment:
Need to sort on _id before skip/limit otherwise several requests for the same page will end up with different results.